frontend/src/component/common/ConstraintAccordion/ConstraintAccordionEdit/ConstraintAccordionEditBody/DateSingleValue/DateSingleValue.tsx

```diff
@@ -1,41 +1,116 @@
 import { ConstraintFormHeader } from '../ConstraintFormHeader/ConstraintFormHeader';
 import Input from 'component/common/Input/Input';
 import { parseDateValue, parseValidDate } from 'component/common/util';
+import {
+    ITimezoneSelect,
+    useTimezones,
+} from '../../../../../../hooks/useTimezones';
+import { utcToZonedTime, zonedTimeToUtc } from 'date-fns-tz';
+
+import { useEffect, useMemo, useState } from 'react';
+import GeneralSelect from '../../../../GeneralSelect/GeneralSelect';
+import { ConditionallyRender } from '../../../../ConditionallyRender/ConditionallyRender';
+import useUiConfig from '../../../../../../hooks/api/getters/useUiConfig/useUiConfig';
+import { styled } from '@mui/material';
+
 interface IDateSingleValueProps {
     setValue: (value: string) => void;
     value?: string;
     error: string;
     setError: React.Dispatch<React.SetStateAction<string>>;
 }
 
+const StyledWrapper = styled('div')(({ theme }) => ({
+    display: 'flex',
+    flexDirection: 'row',
+    marginBottom: 8,
+    alignItems: 'center',
+    gap: theme.spacing(1),
+}));
+
 export const DateSingleValue = ({
     setValue,
     value,
     error,
     setError,
 }: IDateSingleValueProps) => {
+    const { uiConfig } = useUiConfig();
+    const showSelectableTimezone = Boolean(uiConfig.flags.selectableTimezone);
+    const timezones = useTimezones();
+    const { timeZone: local } = Intl.DateTimeFormat().resolvedOptions();
```

Comment:
I was a bit unsure where the `local` variable was defined. It also wasn't clear what it was, so maybe renaming it to `localTimeZone` would help?

```suggestion
    const { timeZone: localTimeZone } = Intl.DateTimeFormat().resolvedOptions();
```

But on further inspection, I see that `localTimezone` is another variable that's defined. But that's only ever used in cases where `showSelectableTimezone` is false 🤔